mx.truffleruby/suite.py

```diff
@@ -116,11 +116,29 @@
             ],
         },
 
+        "truffleruby-shared": {
+            "dir": "src/shared",
+            "sourceDirs": ["java"],
+            "dependencies": [
+                "truffleruby:TRUFFLERUBY-ANNOTATIONS",
+                "sdk:GRAAL_SDK",
+            ],
+            "annotationProcessors": [
+                "TRUFFLERUBY-PROCESSOR",
+            ],
+            "javaCompliance": "1.8",
+            "workingSets": "TruffleRuby",
+            "checkPackagePrefix": "false",
+            "license": [
+                "EPL-1.0",          # JRuby (we're choosing EPL out of EPL,GPL,LGPL)
+            ],
+        },
+
         "truffleruby-processor": {
             "dir": "src/processor",
             "sourceDirs": ["java"],
             "dependencies": [
-                "truffleruby-annotations",
+                "truffleruby:TRUFFLERUBY-ANNOTATIONS",
```

Comment:
Since it's in our own suite.py, I don't think we need a `truffleruby:` prefix.
But it's probably fine to have it too, just seems redundant.